pkg/api/v1/control-plane/server/generic_server.go

```diff
@@ -222,7 +222,7 @@ func (s *server) StreamV2(
 type sendFunc func(resp cache.Response, typeURL string, streamId int64, streamNonce *int64) (string, error)
 
 func (s *server) sendV2(
-	stream envoy_service_discovery_v2.AggregatedDiscoveryService_StreamAggregatedResourcesServer,
+	stream gloo_discovery.GlooDiscoveryService_StreamAggregatedResourcesServer,
```

Comment:
I find it slightly confusing that we have a v2 (envoy) method expecting a gloo discovery service stream. I know that the GlooDS is based on v2 envoy protos, though others will have to remember that context. Similar to what I asked above, do you think it's worthwhile to make the server implement StreamGlooDS and sendGlooDS methods so that it's clear their purpose is distinct?